integration_test/third_party_apps_data/applications/hbase/sles/post

```diff
@@ -0,0 +1,4 @@
+set -e
+
+sudo systemctl restart hbase 
+sleep 30
```

Comment:
For a metrics side only PR, the Fluent Bit submodule should not be updated.

![image](https://user-images.githubusercontent.com/5287526/152095161-dd82fb81-e0b4-4ce1-90d9-dab24ea162ea.png)

Reply:
Changes Removed